client/src/components/Carousel.jsx

```diff
@@ -1,13 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import Card from './Card.jsx';
+import axios from 'axios';
+
+const Carousel = ({ productId, cardType }) => {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    // if cardType is related
+    axios.get('/products/22122/related')
```

Comment:
What is this "magic number' of 22122? Is that something we could put in a const above to clarify what it is?

Reply:
Sounds good.